api/v3/Payment.php

```diff
@@ -41,6 +41,9 @@
  *   Array of financial transactions which are payments, if error an array with an error id and error message
  */
 function civicrm_api3_payment_get($params) {
+  if (!CRM_Core_Permission::check('access CiviContribute')) {
```

Comment:
Thanks! I didn't know that. So if I set the required permissions there, will the system still check for the permissions even if the check_permissions flag is not specified?